icu4c/source/test/perf/DateFmtPerf/DateFmtPerf.vcxproj

```diff
@@ -14,18 +14,19 @@
     <ProjectGuid>{B5E59422-0C52-4469-AB69-0C2AFD6E37A5}</ProjectGuid>
     <RootNamespace>DateFmtPerf</RootNamespace>
     <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>10.0.18288.0</WindowsTargetPlatformVersion>
```

Comment:
So rather than hard-code a particular version of the platform in every single file, I'm wondering if it might be better to refer to the shared build props file that is used by the other vcxproj files?

I extracted various common settings and put them in a shared file: `Build.Windows.ProjectConfiguration.props`.
https://github.com/unicode-org/icu/blob/master/icu4c/source/allinone/Build.Windows.ProjectConfiguration.props

For example, the `date` sample app pulls it in here:
https://github.com/unicode-org/icu/blob/master/icu4c/source/samples/date/date.vcxproj#L12

This would actually let us remove the `WindowsTargetPlatformVersion` and `PlatformToolset` nodes from the vcxproj files. :)

Reply:
Thanks for the feedback Jeff, this would definitely be better to do!